Source/CarthageKit/Project.swift

```diff
@@ -745,10 +771,13 @@ public final class Project { // swiftlint:disable:this type_body_length
 			}
 
 		return loadResolvedCartfile()
-			.flatMap(.merge) { resolvedCartfile in
-				return self
-					.buildOrderForResolvedCartfile(resolvedCartfile, dependenciesToInclude: dependenciesToCheckout)
-					.collect()
+			.map { resolvedCartfile -> [(Dependency, PinnedVersion)] in
+				let dependencies = resolvedCartfile.dependencies
+				return dependencies.keys
```

Comment:
Since we were already doing checkouts concurrently, I realized we were wasting cycles figuring out the build order, and we really just want to checkout anything we find in the list.